app/app/src/main/java/com/globant/calculator/android/mvp/view/ActivityView.java

```diff
@@ -1,5 +1,32 @@
 package com.globant.calculator.android.mvp.view;
 
+import android.app.Activity;
+import android.app.FragmentManager;
+import android.content.Context;
+import androidx.annotation.Nullable;
+import java.lang.ref.WeakReference;
+
 public class ActivityView {
+    private WeakReference<Activity> activityRef;
+
+    public ActivityView(Activity activity) {
+        activityRef = new WeakReference<>(activity);
+    }
+
+    @Nullable
+    public Activity getActivity() {
+        return activityRef.get();
+    }
+
+    @Nullable
+    public Context getContext() {
```

Comment:
this method is needed?

Reply:
that is the basic ActivityView that we provide them.

Reply:
ok, thaks